tests/factories.py

```diff
@@ -718,3 +719,69 @@ def _create(cls, *args, **kwargs):
             )
         )
         return super(NodeLicenseRecordFactory, cls)._create(*args, **kwargs)
+
+
+def render_generations_from_parent(parent, creator, num_generations):
+    current_gen = parent
+    for generation in xrange(0, num_generations):
+        next_gen = NodeFactory(
+            parent=current_gen,
+            creator=creator,
+            title=fake.science_sentence(),
```

Comment:
Attempted to address this in https://github.com/CenterForOpenScience/osf.io/pull/5382/commits/64532c5040cf47eb7d3a366c7d733c05b767f74f